src/timepicker/TimePickerColumnValue.ts

```diff
@@ -26,33 +35,39 @@ export const useTimePickerColumnValue = createHook<
   TimePickerColumnValueHTMLProps
 >({
   name: "TimePickerColumnValue",
-  compose: useCompositeItem,
+  compose: [useButton, useCompositeItem],
   keys: TIME_PICKER_COLUMN_VALUE_KEYS,
 
-  useProps(
-    { setCurrentId, move, selected, value, id, onSelection },
-    { ref, ...htmlProps },
-  ) {
-    const htmlRef = React.useRef();
+  useProps(options, { ref: htmlRef, onClick: htmlOnClick, ...htmlProps }) {
+    const {
+      setCurrentId,
+      move,
+      selected,
+      value,
+      id,
+      setSelected,
+      visible,
+    } = options;
+    const ref = React.useRef<HTMLElement>();
+    const valueProp = value === "AM" ? 0 : value === "PM" ? 1 : +value;
```

Comment:
Okay got it, so you are forcing number inputs and casting AM to 0 and PM to 1. 

The column still works with any other value right? 🤔  or not?

![image](https://user-images.githubusercontent.com/35374649/95457873-14de0500-098f-11eb-8a32-8a83f824e595.png)

Reply:
So, what do you say.

This is a bad idea 🤔 , needs a correction or revert back to how it was ?

Reply:
Let's try it out and see what problems it makes with arbitrary values

Reply:
the basic keyboard navigations works, but selection returns NaN

![image](https://user-images.githubusercontent.com/35374649/95461775-42797d00-0994-11eb-9bf5-f23d53ecc8a0.png)
![image](https://user-images.githubusercontent.com/35374649/95461871-5cb35b00-0994-11eb-8277-e024e3f752ef.png)

Reply:
If we use the map index then it works... so the behaviour changes to index based instead of value based.

![image](https://user-images.githubusercontent.com/35374649/95462043-9d12d900-0994-11eb-9f5d-03b39c37e2bc.png)